### ISSUE TITLE
Class Laravel\Telescope\Contracts\EntriesRepository does not exist 

### ISSUE THREAD

**Issue by jroski** [tighten/ziggy#358]
#### Description

When using this package in conjunction with https://github.com/barryvdh/laravel-debugbar, Calling `@routes` blade directive or the artisan command triggers the error:

`Class Laravel\Telescope\Contracts\EntriesRepository does not exist`

#### Environment

- **Laravel version**: 7.x
- **Ziggy version**: 1.0

**Related routes**:

n/a

**Comment by bakerkretzmar** [comment]
Double-check your `composer.json` to make sure you have Telescope installed correctly in the environment you're seeing these errors in. Do you get similar errors running other `artisan` commands (not provided by Ziggy)?

Closing because this doesn't sound like an issue with Ziggy.

**Comment by jroski** [comment]
I do not have Telescope installed. I believe the error is thrown because a route is defined in the debugbar package for Telescope even if it isn't installed. I've located the specific route in the debugbar package:

`
$router->get('telescope/{id}', [
    ' uses' => 'TelescopeController@show',
     'as' => 'debugbar.telescope',
]);
`
The `TelescopeController@show` method has a dependency of `Laravel\Telescope\Contracts\EntriesRepository`, which doesn't exist unless Telescope is installed.

**Comment by jroski** [comment]
Downgrading to 0.9.4 fixes the issue.

**Comment by bakerkretzmar** [comment]
@jroski thanks, I'll work on this. Debugbar registers the Telescope route even if Telescope isn't installed, so when Ziggy tries to reflect its parameter typehints, they don't exist.

**Comment by bakerkretzmar** [comment]
@jroski I actually can't reproduce this with Laravel 7 or 8.

Ziggy only tries to reflect/instantiate route parameters that implement the `UrlRoutable` contract, with Telescope's `EntriesRepository` doesn't, so I'm not sure what's happening here.

Is there additional error output you can provide? Can you share what specific version of Laravel and Debugbar you're on?

Thanks.

**Comment by jroski** [comment]
Laravel version: 7.10.2
Debugbar version: 3.5.1

Stacktrace:
#0 \\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\RouteSignatureParameters.php(25): ReflectionParameter->getClass()
#1 [internal function]: Illuminate\\Routing\\RouteSignatureParameters::Illuminate\\Routing\\{closure}(Object(ReflectionParameter))
#2 \\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\RouteSignatureParameters.php(26): array_filter(Array, Object(Closure))
#3 \\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php(491): Illuminate\\Routing\\RouteSignatureParameters::fromAction(Array, 'Illuminate\\\\Cont...')
#4 \\vendor\\tightenco\\ziggy\\src\\Ziggy.php(152): Illuminate\\Routing\\Route->signatureParameters('Illuminate\\\\Cont...')
#5 \\vendor\\tightenco\\ziggy\\src\\Ziggy.php(94): Tightenco\\Ziggy\\Ziggy->resolveBindings(Array)
#6 \\vendor\\tightenco\\ziggy\\src\\Ziggy.php(25): Tightenco\\Ziggy\\Ziggy->nameKeyedRoutes()
#7 \\vendor\\tightenco\\ziggy\\src\\BladeRouteGenerator.php(11): Tightenco\\Ziggy\\Ziggy->__construct(false)
#8 \resources\\views/inertia.blade.php(23): Tightenco\\Ziggy\\BladeRouteGenerator->generate()

**Comment by bakerkretzmar** [comment]
@jroski thanks—this was fixed in Laravel 7.14.0, updating will resolve it. See https://github.com/laravel/framework/commit/1af06328c77b875dfb3304f3e9611c2078314acb.

**Comment by Sogl** [comment]
Just updated ziggy from 0.7.1 to 1.1.0 and `php artisan ziggy:generate` show me this output:
```php

laradock@34e3005574a2:/var/www/sogl$ php artisan ziggy:generate

   ReflectionException  : Class Laravel\Telescope\Contracts\EntriesRepository does not exist

  at /var/www/sogl/vendor/laravel/framework/src/Illuminate/Routing/RouteSignatureParameters.php:25
    21|                         ? static::fromClassMethodString($action['uses'])
    22|                         : (new ReflectionFunction($action['uses']))->getParameters();
    23| 
    24|         return is_null($subClass) ? $parameters : array_filter($parameters, function ($p) use ($subClass) {
  > 25|             return $p->getClass() && $p->getClass()->isSubclassOf($subClass);
    26|         });
    27|     }
    28| 
    29|     /**

  Exception trace:

  1   ReflectionParameter::getClass()
      /var/www/sogl/vendor/laravel/framework/src/Illuminate/Routing/RouteSignatureParameters.php:25

  2   Illuminate\Routing\RouteSignatureParameters::Illuminate\Routing\{closure}(Object(ReflectionParameter))
      [internal]:0

  Please use the argument -v to see more details.
```

I'm on Laravel 5.8, PHP 7.2, I don't use Telescope.
@bakerkretzmar Only updating to Laravel 7 can help?

**Comment by Sogl** [comment]
Solved after update to PHP 7.3, Laravel 6.20.27, laravel-debugbar 3.5.7
